src/error_handling.rs

```diff
@@ -2,7 +2,7 @@ use log::error;
 
 fn show_error(explanation: &str) -> ! {
     error!("{}", explanation);
-    panic!(explanation.to_string());
+    panic!("{}", explanation.to_string());
```

Comment:
This feels dumb. Do you know of a way to disable a given clippy suggestion? Or otherwise can you give some details as to why this was suggested? It doesn't make sense to me.

Reply:
```
warning: panic message is not a string literal
 --> src/error_handling.rs:5:12
  |
5 |     panic!(explanation.to_string());
  |            ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(non_fmt_panic)]` on by default
  = note: this is no longer accepted in Rust 2021
help: add a "{}" format string to Display the message
  |
5 |     panic!("{}", explanation.to_string());
  |            ^^^^^
help: or use std::panic::panic_any instead
  |
5 |     std::panic::panic_any(explanation.to_string());
```
I didn't want to use `panic_any` ([here](https://doc.rust-lang.org/std/panic/fn.panic_any.html)) becaus this would be surely be string

Reply:
Makes good sense!